package.json

```
@@ -49,7 +49,8 @@
     "test.testing": "node scripts/test/validate-testing.js",
     "test.sys.node": "echo test.sys.node",
     "test.prod": "npm run test.dist && npm run test.end-to-end && npm run test.jest && npm run test.karma && npm run test.sys.node && npm run test.testing",
-    "test.watch": "jest --watch"
+    "test.watch": "jest --watch",
+    "test.watchAll": "jest --watchAll --coverage"
```

The convention for this file is to use dash-case instead of camelCase for multi-word scripts. Can we please do something like:
```suggestion
    "test.watch-all": "jest --watchAll --coverage"
```